transit_odp/organisation/querysets.py

```diff
@@ -1152,6 +1153,32 @@ def get_overall_data_catalogue(self):
     def filter_by_noc_and_line_name(self, noc, line_name):
         return self.filter(national_operator_code=noc, line_names__contains=[line_name])
 
+    def add_effective_stale_date_last_modified_date(self):
+        """Adds the Effective Stale Date: Last Modified Date of file + 1 year."""
+        return self.annotate(
+            effective_stale_date_last_modified_date=ExpressionWrapper(
+                F("modification_datetime") + timedelta(days=365),
```

Comment:
Doesn't handle leap years. I guess you're saying that's an acceptable risk?

Reply:
Using `relativedelta` here returns a `can't adapt type relativedelta` error. 

If the intention of the staleness here is to flag that it hasn't been updated for a year, being one day out every 4 years and still flagging as stale seems like an acceptable risk and still encouraging updating of data. 

This will need to be consistent with how it's calculated in the file downloads or we might end up with discrepancies - how have you calculated it for your csv?

Reply:
Well I'm doing it in Python not in the ORM, so I can do:
`datetime.date(d.year+1, d.month, d.day)`

But I doubt that's possible in the ORM.

I'm just flagging it up for the record. If you and Paul have tried to do it and got nowhere, that's fair enough.

Reply:
Recently I was facing the same problem 
`relativedelta` doesn't work in the Query, this sentence:
`F("modification_datetime") + relativedelta(years=+1)` 
it's not allow in Django!
The only type allow is `datetime.timedelta`

```
type(timedelta(days=365)) -> datetime.timedelta 
type(relativedelta(years=+1)) -> dateutil.relativedelta.relativedelta
```